src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
```

Comment:
you can give a name in upperCase if it's some global variables or mock data. So, you can just name in camelCase